scripts/travis-functest.sh

```diff
@@ -12,6 +12,6 @@ sudo scripts/minikube.sh k8s-sidecar
 sudo chown -R travis: "$HOME"/.minikube /usr/local/bin/kubectl
 # functional tests
 
-go test github.com/ceph/ceph-csi/e2e --deploy-timeout=10 -timeout=30m -v
+go test github.com/ceph/ceph-csi/e2e --deploy-timeout=10 -timeout=30m --cephcsi-namespace=cephcsi-testing -v
```

Comment:
may be a variable for value `cephcsi-testing` is better.

Reply:
i don't think we need to define a new variable and use it, I have just followed a pattern which we used earlier.

Reply:
I prefer to use a changing namespace, something like `--cephcsi-namespace=cephcsi-testing-${RANDOM}`. That makes sure the namespace will not get hardcoded in other locations later on.